src/components/button/button.jsx

```diff
@@ -0,0 +1,30 @@
+const classNames = require('classnames');
+const PropTypes = require('prop-types');
+const React = require('react');
+
+const styles = require('./button.css');
+
+const ButtonComponent = ({
+    className,
+    onClick,
+    children,
+    ...props
+}) => (
+    <span
```

Comment:
Not sure if this is important now, but we should probably designate this as a button for ARIA using `role="button"`